src/utils/MSstatsConverter.cpp

```diff
@@ -209,7 +206,6 @@ class TOPPMSstatsConverter final :
     static const String param_reannotate_filenames;
 
 private:
-
     static void fatalErrorIf_(const bool error_condition, const String &message, const int exit_code)
```

Comment:
move this to Exception.h and call it: throwReturnCode ...

question: if the exception is not caught, does the progam actually have the return code thrown?

Reply:
I have no idea ;) not my code